community/detectors/apache_http_server_cve_2021_41773/src/main/java/com/google/tsunami/plugins/detectors/rce/cve202125646/ApacheHttpServerCVE202141773VulnDetector.java

```diff
@@ -0,0 +1,112 @@
+package com.google.tsunami.plugins.detectors.rce.cve202125646;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.tsunami.common.net.http.HttpRequest.get;
+import static com.google.tsunami.common.net.http.HttpRequest.post;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.flogger.GoogleLogger;
+import com.google.inject.Inject;
```

Comment:
Can you import `javax.inject.Inject` instead of this one? Same for the unit test file.

Reply:
updated,done.

Reply:
This file is not updated yet :)